### PR TITLE
Update people page

### PR DESCRIPTION
**Pull request by cdccollins**

### CODE REVIEW

File: data/people.yml [hunk 1]
```diff
@@ -201,10 +215,8 @@
     link: https://www.linkedin.com/in/david-henry-000b603/
   - name: Github
     link: https://github.com/dwhenry
-- name: Celia Collins
```
Comment (h-lame) [inline data/people.yml:204]: Have you inadvertently removed yourself @cdccollins ?

**Comment by ubxd** [comment]
Deploy preview for *unboxed-co-production* ready!

Built with commit ba08379bf6431151e1af348c403015bc9ba1ac43

https://deploy-preview-526--unboxed-co-production.netlify.com

File: data/people.yml (hunk 1 above)
Comment (h-lame) [reply data/people.yml:204]: Oh, no you haven't, you've just moved.  Sorry!

Comment (cdccollins) [reply data/people.yml:204]: I moved myself up 🙃. Graeme wants a woman on each line of people

**Review by h-lame — APPROVED** [review]

**Pull request merged**